### PR TITLE
[changelog] Describe SE-0370

### PR DESCRIPTION
**Pull request by glessard**
Add a changelog entry describing SE-0370.

### CODE REVIEW

**Comment by glessard** [comment]
@swift-ci please build toolchain macOS platform

**Comment by glessard** [comment]
@swift-ci please smoke test

**Comment by glessard** [comment]
@swift-ci please build toolchain macOS platform

**Comment by glessard** [comment]
@swift-ci please smoke test

**Comment by glessard** [comment]
@swift-ci please smoke test

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
     }
   }
   ``` 
+* [SE-0370][]:
+
+  The API of `UnsafeMutableRawPointer`, `UnsafeMutableBufferPointer`, `UnsafeMutableRawBufferPointer` were improved, adding previously missing initialization (and deinitialization) methods, including more performant initialization from `Collection` types.
+
+  For `UnsafeMutablePointer<T>` and `UnsafeMutableBufferPointer<T>`, method names containing the word "assign" were renamed to use the word "update", and many more were added. Every initialization method of `UnsafeMutablePointer` and `UnsafeMutableBufferPointer` has a corresponding "update" method.
```
[the remark this answers is not in the document]
Comment (glessard) [reply CHANGELOG.md:25]: Ah, yes, good point. Will revise.

**Review by benrimmington — APPROVED** [review]

**Comment by glessard** [comment]
@swift-ci please smoke test and merge

**Comment by glessard** [comment]
Thanks @benrimmington!

**Pull request merged**